iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/util/IidmXmlUtil.java

```diff
@@ -21,6 +21,20 @@
  */
 public final class IidmXmlUtil {
 
+    public interface IidmXmlRunnable extends Runnable {
+
+        @Override
+        default void run() {
```

Comment:
You should mark this method final to avoid someone to override it?

Reply:
It seems it is not possible to make methods in interfaces final (even with default implementation)